lucille-core/src/test/java/com/kmwllc/lucille/stage/AddRandomFieldTest.java

```diff
@@ -0,0 +1,110 @@
+package com.kmwllc.lucille.stage;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertThrows;
+import static org.junit.Assert.assertTrue;
+
+import com.kmwllc.lucille.core.Document;
+import com.kmwllc.lucille.core.Stage;
+import com.kmwllc.lucille.core.StageException;
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+
+import com.kmwllc.lucille.stage.StageFactory;
+import java.nio.file.Path;
+import java.util.Arrays;
+import java.util.List;
+import org.junit.Test;
+
+public class AddRandomFieldTest {
+
+  private final StageFactory factory = StageFactory.of(AddRandomField.class);
+
+  /**
```

Comment:
would be ideal to describe what this test is doing
however, if not planning to do that, this empty comment should be removed

Reply:
added comments for all test methods